packages/dialtone-css/lib/build/less/components/item-layout.less

```diff
@@ -13,41 +13,79 @@
 //  $   BASE STYLE
 //  ----------------------------------------------------------------------------
 .dt-item-layout {
-  display: flex;
-  align-items: center;
-  min-height: calc(var(--dt-size-550) + var(--dt-size-300));
-  padding: var(--dt-space-300) var(--dt-space-400);
-  font-size: var(--dt-font-size-200);
-  line-height: var(--dt-font-line-height-300);
+  display: grid;
+  grid-template-areas:
+    'left content right selected';
+  grid-template-columns: repeat(4, minmax(0, max-content));
 
-  &--content {
-    flex-grow: 1;
-    min-width: var(--dt-size-0);
-  }
+  &:not(&--custom) {
+    grid-template-columns: minmax(0, max-content) minmax(0, 1fr) minmax(0, max-content) minmax(0, max-content);
+    align-items: center;
+    min-height: calc(var(--dt-size-550) + var(--dt-size-300));
+    padding: var(--dt-space-300) var(--dt-space-400);
+    font-size: var(--dt-font-size-200);
+    line-height: var(--dt-font-line-height-300);
 
-  &--subtitle {
-    color: var(--dt-color-foreground-tertiary);
-    font-size: var(--dt-font-size-100);
+    .dt-item-layout--subtitle {
+      color: var(--dt-color-foreground-tertiary);
+      font-size: var(--dt-font-size-100);
+
+      &--with-title {
+        margin-top: var(--dt-space-200-negative);
+      }
+    }
+
+    .dt-item-layout--selected {
+      align-self: center;
+    }
+
+    .dt-item-layout--left {
+      padding-right: var(--dt-space-400);
```

Comment:
Would it make sense to use gap instead of padding? The only cell that doesn't have this padding is selected and maybe it should have, for what I see in the use in list-item they are adding a margin:
<img width="379" alt="image" src="https://github.com/dialpad/dialtone/assets/24460973/b7bb43ce-cb20-4ced-ad7a-64fd44955035">

Reply:
Yes, makes sense to use gap instead of paddings, just migrated the old stylings, forgot to update them accordingly.